specification/compute/resource-manager/Microsoft.Compute/stable/2021-03-01/compute.json

```diff
@@ -3247,11 +3250,12 @@
             "type": "string",
             "description": "The expand expression to apply on the operation.",
             "enum": [
-              "instanceView"
+              "instanceView",
+              "userData"
             ],
             "x-ms-enum": {
               "name": "InstanceViewTypes",
-              "modelAsString": false
+              "modelAsString": true
```

Comment:
Why is this being changed?

Reply:
I wonder if this change is causing the other .NEt SDK PR check to fail. Please investigate!

Reply:
@Sandido I'm changing from false to true because you instructed me to do that =)

Reply:
@roipmsft , Hm, perhaps there was a misunderstanding. 
Enums that were previously modelAsString False should remain that way. New ones should be True.

Reply:
change enum type name in stable API version will cause breaking change in SDKs. Pls revert the name changing part if not necessary.

Reply:
This is necessary, unfortunately.
The way that the current existing Compute.json implements these enums is flat-out wrong.

These enums have to be correct at some point. Surely, this incorrect approach cannot remain in the Swagger forever?

I run this by the BreakingChange Review folks
https://msazure.visualstudio.com/One/_workitems/edit/9701056